packages/material-ui-lab/src/TreeItem/TreeItemContent.js

```diff
@@ -1,13 +1,48 @@
 import * as React from 'react';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
+import { experimentalStyled } from '@material-ui/core/styles';
 import useTreeItem from './useTreeItem';
 
+const TreeItemContentIconContainer = experimentalStyled(
```

Comment:
See the previous comment on the changes of the demos. We should move these under the `TreItemContent` slot component.

Reply:
done